src/__tests__/proposalVersions.int.test.ts

```diff
@@ -0,0 +1,1272 @@
+import request from 'supertest';
+import { TinyPgError } from 'tinypg';
+import { app } from '../app';
+import { db } from '../database';
+import { getLogger } from '../logger';
+import {
+  getTableMetrics,
+  isoTimestampPattern,
+} from '../test/utils';
+import { PostgresErrorCode } from '../types/PostgresErrorCode';
+import type { Result } from 'tinypg';
+
+const logger = getLogger(__filename);
+const agent = request.agent(app);
+
+describe('/proposalVersions', () => {
+  describe('POST /', () => {
+    it('creates exactly one proposal version', async () => {
+      await db.query(`
+        INSERT INTO opportunities (
+          title,
+          created_at
+        )
+        VALUES
+          ( '🔥', '2525-01-02T00:00:01Z' )
```

Comment:
I never mentioned it before, and it is definitely not a blocker, but I think it helps to have completely distinct values in test data such that if we are looking at a failed test with db state having a value there is exactly one place in the test code that could have produced that value. Maybe on a rainy day after we have much more important things accomplished I might make updates to this effect.

Reply:
This is a good suggestion (appreciate the note to not block on it for now, as I agree it would be worth fixing en masse)

Reply:
Captured in https://github.com/PhilanthropyDataCommons/service/issues/108